product_docs/docs/transporter/ba_preview/limitations.mdx

```diff
@@ -19,6 +19,6 @@ Unsupported Oracle data types include:
 - XML
 - Spatial
 
-Don't use EDB Transporter for replicating Oracle tables that contain blob, clob, or nclob columns and are expected to get UPDATE changes. Streaming UPDATE changes currently aren't handled correctly.
+Don't use EDB Transporter for replicating Oracle tables that contain BLOB, CLOB, or NCLOB columns that are expected to change. These types are supported for snapshots but are not captured in the streaming CDC replication.
```

Comment:
@yidian1997 did I get this right?

Reply:
Here is a chart I added to our team doc. We support streaming LOB for tables containing **primary key**. [Jira Issue](https://enterprisedb.atlassian.net/browse/ET-335)
For tables that don't have PK and have LOB type, we only support insert op.